docusaurus/docs/adding-typescript.md

```diff
@@ -12,14 +12,16 @@ title: Adding TypeScript
 To start a new Create React App project with [TypeScript](https://www.typescriptlang.org/), you can run:
 
 ```sh
-npx create-react-app my-app --typescript
+npx create-react-app my-app --template typescript
 
 # or
 
-yarn create react-app my-app --typescript
+yarn create react-app my-app --template typescript
 ```
 
 > If you've previously installed `create-react-app` globally via `npm install -g create-react-app`, we recommend you uninstall the package using `npm uninstall -g create-react-app` to ensure that `npx` always uses the latest version.
+>
+> Global installs of `create-react-app` are no longer supported.
```

Comment:
I mentioned this in the Discord channel a while back but I am kinda worried about this change: it needs to be done but so many online guides start by installing `create-react-app` globally. I think it might be useful to document this somewhere (well, at least we need to have it in the migration guide) and link it from eg.  https://github.com/facebook/create-react-app/pull/7716/files#diff-0ae46a2383dedcac5f5e369f5ca5f169R95-R98